apps/passport-server/src/services/issuanceService.ts

```diff
@@ -522,6 +624,84 @@ export class IssuanceService {
     }
   }
 
+  /**
+   * Checks that a ticket is valid for check-in based on the ticket data in
+   * the DB.
+   */
+  public async handleCheckTicketByIdRequest(
```

Comment:
worth communicating both in comment and in name of function that this is only for deconnect tickets